src/OpenSage.Game/Gui/Wnd/Controls/ListBox.cs

```diff
@@ -289,17 +300,36 @@ public int MaxDisplay
             }
         }
 
-        public void UpdateSelectedItem()
+        private void SetSelectedItem()
```

Comment:
Since this is not a setter method (and doesn't take a parameter) I think the previous name was better.

When I see a call to `SetSelectedItem()` without an argument, the first question that comes to my mind is "Set it to what?"